.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
   - "0.10"
 before_script:
   - cp config.js.example config.js
+  - npm run nsp
```

Comment:
Thanks, Jérémie. I'll put it *inside* `script` then soon, if @deniak is OK with that too. It seems like a good compromise.